src/index.html

```diff
@@ -5,10 +5,54 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header-container">
+        <a href="#home" class="logo__link">
+          <img
+            src="./images/logo.png"
+            alt="logo_moyo"
+            class="logo__img"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="item__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="item__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="item__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="item__link"
+                data-qa="hover">Laptops & Computers</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="item__link">Gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="item__link">Tablets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="item__link">Photo</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="item__link">Video</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__item">
              <a href="#" class="item__link">Gadgets</a>
            </li>
            
            <li class="nav__item">
              <a href="#" class="item__link">Tablets</a>
            </li>
            
            <li class="nav__item">
              <a href="#" class="item__link">Photo</a>
            </li>
            
            <li class="nav__item">
              <a href="#" class="item__link">Video</a>
            </li>
```
add spaces between multiline siblings